pyntc/devices/asa_device.py

```diff
@@ -605,6 +675,12 @@ def open(self):
 
     @property
     def peer_device(self) -> "ASADevice":
+        """
+        Create instance of ASADevice for peer device.
+
+        Returns:
+            CiscoASA: Instance of CiscoASA class.
```

Comment:
If we want to use Sphinx documentation, I would use this format for classes:
:class:`~<package>.<ClassName>`: Description.
In this case it would be:
:class:`~devices.ASADevice`: Cisco ASA device instance.

Reply:
Did not know about this. Thanks for letting me know. This has been updated and I'll keep that in mind going forward.